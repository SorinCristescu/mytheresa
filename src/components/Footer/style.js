import styled from 'styled-components';

export const FooterContainer = styled.div`
  width: 100%;
  position: fixed;
  bottom: 0;
  z-index: 10000;
  display: flex;
  align-items: center;
  justify-content: center;

  span {
    color: grey;
    text-align: center;
    font-size: 12px;
  }
`;
