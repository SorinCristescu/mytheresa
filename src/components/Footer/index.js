import React from 'react';
import PropTypes from 'prop-types';
import { FooterContainer } from './style';

const Footer = () => {
  return (
    <FooterContainer>
      <span>Built with passion by STC</span>
    </FooterContainer>
  );
};

export default Footer;
