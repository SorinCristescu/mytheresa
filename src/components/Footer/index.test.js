import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

it('to render Footer component', () => {
  expect(shallow(<Footer />)).toMatchSnapshot();
});
