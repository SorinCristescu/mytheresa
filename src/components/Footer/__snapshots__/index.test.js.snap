// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to render Footer component 1`] = `
<styled.div>
  <span>
    Built with passion by STC
  </span>
</styled.div>
`;
